sdk/src/types/block/protocol.rs

```diff
@@ -285,7 +287,6 @@ impl Default for CongestionControlParameters {
     }
 }
 
-// TODO docs
```

Comment:
but no docs :c

Reply:
Oh sorry, removed it because of the fields but forgot the main one, will add later

Reply:
Done